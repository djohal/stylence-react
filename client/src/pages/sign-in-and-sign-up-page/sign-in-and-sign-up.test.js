import React from "react";
import { shallow } from "enzyme";
import SignInAndSignUpPage from "./sign-in-and-sign-up.component";

it("should render sign-in-and-sign-up component", () => {
  expect(shallow(<SignInAndSignUpPage />)).toMatchSnapshot();
});
