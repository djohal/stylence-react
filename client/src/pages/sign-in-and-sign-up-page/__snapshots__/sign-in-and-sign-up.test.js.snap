// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render sign-in-and-sign-up component 1`] = `ShallowWrapper {}`;
